app/reducers/MomentReducer.js

```diff
@@ -0,0 +1,19 @@
+import { MOMENT_ACTIONS } from '../constants';
+import SessionUtils from '../utils/SessionStorage';
+import update from 'react-addons-update';
+
+class MomentReducer {
+  moment(state = [], action) {
+    switch (action.type) {
+      case MOMENT_ACTIONS.likeSuccess:
+        console.log('moment response: ', action.payload.data);
```

Comment:
Yes Sir, i'm yet to update the state with the payload data, thus the debug code. I sent you a mail regarding the challenge i'm facing with this